backend/connectors/rasa-connector/src/main/java/co/airy/core/rasa_connector/RasaClientConfig.java

```diff
@@ -0,0 +1,24 @@
+package co.airy.core.rasa_connector;
+
+
+import feign.Feign;
+import feign.jackson.JacksonDecoder;
+import feign.jackson.JacksonEncoder;
+import feign.okhttp.OkHttpClient;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class RasaClientConfig {
+    @Bean
+    public RasaClient rasaClient(@Value("${rasa.rest-webhook-url}") String rasaRestUrl) {
+        return Feign.builder()
+                .client(new OkHttpClient())
+                .encoder(new JacksonEncoder())
+                .decoder(new JacksonDecoder())
+                .logger(new feign.Logger.ErrorLogger())
+                .logLevel(feign.Logger.Level.FULL)
+                .target(RasaClient.class, rasaRestUrl);
```

Comment:
There is no authentication token? How do you authenticate witht he rasa api?

Reply:
So to just call the REST API endpoint, no authentication is needed. You need authentication when calling other endpoints like /models.parse etc. (i.e for a human-handoff like feature) but not for getting plain responses (without any other data).